src/c_api_common.h

```diff
@@ -34,13 +34,13 @@ dgl::runtime::PackedFunc ConvertNDArrayVectorToPackedFunc(
  *
  * The element type of the vector must be convertible to int64_t.
  */
-template<typename DType>
+template<typename IdType, typename DType>
```

Comment:
We should delete this function. There is another one called `aten::VecToIdArray` in the header file.

Reply:
This func is not used in the new version. Maybe deleted later when deprecating DGLGraph.